Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -76,6 +76,51 @@ class AppStoreConnectService {
             .execute(with: requestor)
             .awaitMany()
     }
+
+    func listBetaTesters(
+        email: String?,
+        firstName: String?,
+        lastName: String?,
+        inviteType: BetaInviteType?,
+        apps: [String],
+        groupNames: [String],
+        sort: ListBetaTesters.Sort?,
+        limit: Int?,
+        relatedResourcesLimit: Int?
+    ) throws -> [BetaTester] {
+
+        var appIds: [String] = []
+        if !apps.isEmpty {
+            appIds = try GetAppsOperation(options: .init(bundleIds: apps))
+                .execute(with: requestor)
+                .await()
+                .map(\.id)
+        }
+
+        var groupIds: [String] = []
+        if !groupNames.isEmpty {
+            groupIds = try groupNames.map {
+                try betaGroupIdentifier(matching: $0).await()
```

Comment:
I should implement the name filter on `ListBetaGroupsOperation`
Otherwise this is n many synchronous API calls instead of 1